component/loki/source/gcplog/internal/gcplogtarget/push_target.go

```diff
@@ -79,39 +79,39 @@ func (p *PushTarget) push(w http.ResponseWriter, r *http.Request) {
 	bs, err := io.ReadAll(r.Body)
 	if err != nil {
 		p.metrics.gcpPushErrors.WithLabelValues("read_error").Inc()
-		level.Warn(p.logger).Log("msg", "failed to read incoming gcp push request", "err", err.Error())
+		_ = level.Warn(p.logger).Log("msg", "failed to read incoming gcp push request", "err", err.Error())
```

Comment:
Nitpicky comment, but could we revert all these cosmetic changes from commit https://github.com/grafana/agent/pull/3872/commits/50d877ff7c2056435ddfbf547088ae991dee2d3c?

These are more IDE-specific options, and we don't enforce them in our codebase right now; to me it just makes the code a little harder to read.

Reply:
No problem, it's exactly why I've push this in separate commit. I've revert this change